tests/flamegraph.rs

```diff
@@ -63,7 +63,11 @@ fn test_flamegraph_multiple_files(
     }
     // and then compare
     result.set_position(0);
-    compare_results(result, expected, expected_result_file);
+    if std::env::var("FLAMEGRAPH_BLESS").is_ok() {
```

Comment:
Oh, yeah, this is a good idea! I wonder if it should be called `INFERNO_BLESS_TESTS` instead though. Also, would you mind updating `compare_results` so that it mentions this envvar if a test output file doesn't match the stored one?

Reply:
Renamed to `INFERNO_BLESS_TESTS` and added a msg/hint as asked.